gnocchi/tests/test_incoming.py

```diff
@@ -58,6 +58,7 @@ def _iter_on_sacks_to_process():
                 break
             # NOTE(jd) Retry to send measures. It cannot be done only once as
             # there might be a race condition between the threads
+            self.incoming.finish_sack_processing(sack_to_find)
```

Comment:
just curious, why we do this?

Reply:
It's possible another test uses that same sack, and therefore that the flag in Redis is already set for this. Since there's no metricd in those tests, nobody ever call `finish_sack_processing` so set the `setnx` call never send any notification.
By forcing this to be called and re-adding measure, we're sure that at some point this test will get a notification.